### ISSUE TITLE
Add a parameter to install command, so that if it is already installed it won't reinstall

### ISSUE THREAD

**Issue by tgjorgoski** [99xt/serverless-dynamodb-local#59]
Running the `sls dynamodb install` command downloads and installs the dynamodb-local every time even if it is already installed.
It would be nice to have a parameter ( `--no-reinstall` ?)  , in which case the install command would first check if dynamodb-local is installed, and if it is return straight away.

This would help usage of serverless-dynamodb-local in different automation scenarios.

**Comment by AshanFernando** [comment]
@tgjorgoski What if  `sls dynamodb install` internally checks whether its already downloaded and proceed(If already available)? 
We have  `sls dynamodb remove` to remove and install again if someone needs a fresh install

**Comment by tgjorgoski** [comment]
It's weird, I actually tried it again, and it says:
Dynamodb is already installed on path!

I could have sworn it was going on with the download and installation even I got it installed once.
I will close for now, and reopen if I encounter the issue again.

**Comment by AshanFernando** [comment]
Hi, You are right, it downloads and install all the time. Sorry, may be my
previous comment is misleading. I wanted to ask you whether, if 'install'
command intelligently download only once(without parameters), will it solve
your problem?

**Comment by tgjorgoski** [comment]
@AshanFernando , now that I'm trying it, it seems that it actually doesn't download and install all the time -   It downloads and installs the first time, and after that it says "Dynamodb is already installed on path". 

I'm not sure why it was happening that it would reinstall multiple times when I tried it in the past. Maybe it downloads/reinstalls if it is started from different folders?
